t/plugin/referer-restriction.t

```diff
@@ -173,7 +173,6 @@ hello world
             local cases = {
                 "x.*",
                 "~y.xn",
-                "::1",
```

Comment:
The `::1` is an invalid referer.

Reply:
In fact, referer is a URL, and the current shcema check is not suitable. Maybe we need another PR to fix this